front/inventory.php

```diff
@@ -81,6 +90,9 @@
     $redirect_url = $refused->handleInventoryRequest($inventory_request);
     Html::redirect($redirect_url);
 } else {
+    if (isCommandLine()) {
```

Comment:
I think you should put this check before `if (isset($_GET['refused'])) {` as this later check as no sens while using commandline import.